apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -65,6 +65,7 @@ The `create-block-app` package provides everything you need to develop a block.
       - `examples`: an array of example data structures your block would accept and use.
       - `image`: a preview image showing your block in action.
       - `icon`: an icon to be associated with your block.
+      - `name`: a slugified name for your block (which may differ to the package `name` in package.json); it can be defined as `blockname` or `@namespace/blockname`
```

Comment:
This suggestion would apply to the templates too, if you agree with something like it
```suggestion
      - `name`: a slugified name for your block (which may differ to the package `name` in package.json); it can be defined as `blockname` or `@namespace/blockname`, where `namespace` must be your username on blockprotocol.org if you intend to publish it there
```

Reply:
https://github.com/blockprotocol/blockprotocol/pull/953/commits/c16bb7574cdacceab3fd379041e4be7525b95204